frontend/test/metabase/scenarios/native/reproductions/18418-saved-question-db-appears-in-db-picker.cy.spec.js

```diff
@@ -31,6 +35,6 @@ describe("issue 18418", () => {
     // Clicking native question's database picker usually opens a popover with a list of databases
     // As default Cypress environment has only the sample database available, we expect no popup to appear
     cy.findByTextEnsureVisible("Sample Database").click();
-    popover().should("not.exist");
+    cy.get(POPOVER_ELEMENT).should("not.exist");
```

Comment:
This is a trade-off that isn't ideal, but if this helps reduce flakiness so be it.